README.md

```diff
@@ -16,8 +16,7 @@ $ cd grtp.co
 $ make run
 ```
 
-Then visit http://localhost:9537/test/
-
+Then visit [http://localhost:9537/test/](http://localhost:9537/test/)
```

Comment:
I think the one thing I would suggest changing is the link here to use `<>` notation:

`<http://localhost:9537/test/>` -> <http://localhost:9537/test/>

Since it's documentation, and it's only a minor change, this can be done after the merge, so I'm not going to block on it because that would just be silly :P